wmn-data.json

```diff
@@ -5177,6 +5177,16 @@
         "known" : ["spicerex", "rod-it"],
         "cat" : "tech"
        },
+      {
+        "name" : "SPOJ",
+        "uri_check" : "https://www.spoj.com/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "<li>Profile</li>",

```

Comment:
I'm not seeing this `e_string` in the response. Please consider changing it to something like `user-profile-tables`